buildconfigs/oak_tensorflow_enclave_app.toml

```diff
@@ -0,0 +1,4 @@
+# Unlike other build configs, this one needs to run in the "default" shell instead of just the
+# "rust" one, because bazel is also used by the build.rs script.
+command = ["nix", "develop", ".#containers"]
```

Comment:
If you just run this, it will probably hang forever in the shell (it's like just calling `bash`). Try replacing the command with something like `echo hello` for now perhaps